index.html

```diff
@@ -15,16 +15,26 @@
         <div class="toolbar">
           <a href="">John Doe</a>
           <i><img src="./assets//menu.png" /></i>
+          <nav class="menu">
+            <ul>
+              <li><a href="#Portfolio">Portfolio</a></li>
+              <li><a href="#About">About</a></li>
+              <li><a href="#Contact">Contact</a></li>
+            </ul>
+            <img src="./assets/letter.png" />
+          </nav>
         </div>
       </header>
       <section class="headline">
-        <span class="title">Hey there. <br />I'm Alex</span>
-        <span class="sub-title">I'am a software Developer</span>
-        <p>
-          I can help you build a product , feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don’t hestiate to contact me.
-        </p>
+        <div class="infos">
+          <h2 class="title">Hey there. <br />I'm Alex</h2>
```

Comment:
It looks great! but the header section is supposed to occupy the entire visible screen, in other words, it must have `100%` height. So please make sure to increase it.